test/runtime/Policies.go

```diff
@@ -1470,6 +1470,87 @@ var _ = Describe("RuntimePolicies", func() {
 				"Unexpected reply traffic to endpoint")
 		})
 	})
+	Context("Init Policy Default Audit Test", func() {
+		BeforeEach(func() {
+			vm.ContainerRm(initContainer)
+			ExpectPolicyEnforcementUpdated(vm, helpers.PolicyEnforcementAlways)
+		})
+
+		AfterEach(func() {
+			vm.ExecCilium("config PolicyAuditMode=Disabled").ExpectSuccess("unable to change daemon configuration")
+			vm.ContainerRm(initContainer).ExpectSuccess("Container initContainer cannot be deleted")
+		})
+
+		It("Init Ingress Policy Default Audit Test", func() {
```

Comment:
nit:
```suggestion
		It("tests ingress", func() {
```

These strings show up in test failure names along with every parent `Context(...)` string, so otherwise it would be mostly duplicate with the outer context. Also ginkgo style tries to encourage english-style "It ... " sentences in these statements.

Reply:
I could see an argument for something like `tests ingress policy audit` here instead, although the fact that this is policy audit is also captured in the context so *shrug.